### PR TITLE
Eyesightcheck

### PR DESCRIPTION
**Pull request by DragonUncaged**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes:  #608 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [ ] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
preview

### CODE REVIEW

**Review by VidhiBhatt01 — CHANGES_REQUESTED** [review]
In small screen sizes, I can't read what this button says. Fix it please. Also it would be great if you can add a readme file and explain in brief about how this project works.

image

**Comment by VidhiBhatt01** [comment]
https://deploy-preview-620--amazing-js-projects.netlify.app/eyesight%20check/

**Review by DEEPALI-SUK — CHANGES_REQUESTED** [review]
just add one preview.png

**Comment by DragonUncaged** [comment]
> just add one preview.png

added in readme file

**Review by astha2412 — APPROVED** [review]

File: style_links.json [hunk 1]
```diff
@@ -490,5 +490,9 @@
   {
      "name":"LyricZ - Website",
     "link":"Lyricz API lyrics finder/index.html"
-  }
+  },
+    {
+     "name":"Screen distance Measure",
+    "link":"Screen distance Measure/index.html"
```
Comment (arpit456jain) [inline style_links.json:496]: wrong name of project

Comment (DragonUncaged) [reply style_links.json]: Done !!.

**Pull request merged**